<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sJsCalculate</class>
 <widget class="QWidget" name="sJsCalculate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>公式</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>警报阈值下限（N为无限制）</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>警报阈值上限（N为无限制）</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>N2/H2O</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>M28/M18</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>N</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>N</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>O2/H2O</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>M32/M18</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>N</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>N</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>N2/Ar</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>M28/M40</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>N</string>
      </property>
     </item>
     <item row="2" column="3">
      <property name="text">
       <string>N</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>告警设置</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="UI_RB_DurationEX_JSCALC">
          <property name="text">
           <string>超阈值持续时间(S)：</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="UI_LE_DurationEX_JSCALC">
          <property name="text">
           <string>600</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="UI_RB_COUNTS_JSCALC">
          <property name="text">
           <string>超阈值连续点数：</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="UI_LE_COUNTS_JSCALC">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="UI_PB_OK_JSCALC">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>56</height>
        </size>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="UI_PB_CANCLE_JSCALC">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>56</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
